tools/model_tools/src/openvino/model_zoo/quantize_engine/quantizer.py

```diff
@@ -92,20 +92,28 @@ def get_pot_cmd_prefix(self) -> List[str]:
         return pot_cmd_prefix
 
 
-    def quantize(self, reporter, model, precision, target_device, pot_cmd_prefix, pot_env) -> bool:
+    def quantize(self, reporter, model, precision, target_device, pot_cmd_prefix, pot_env, model_root=None) -> bool:
         input_precision = _common.KNOWN_QUANTIZED_PRECISIONS[precision]
 
-        pot_config_base_path = _common.MODEL_ROOT / model.subdirectory / 'quantization.yml'
+        model_root = _common.MODEL_ROOT if model_root is None else model_root
+        pot_config_base_path = model_root / model.subdirectory / 'quantization.yml'
 
         try:
             with pot_config_base_path.open('rb') as pot_config_base_file:
                 pot_config_base = yaml.safe_load(pot_config_base_file)
+
         except FileNotFoundError:
+            reporter.log_warning(f"Unable to locate quantization.yml in {pot_config_base_path}, " +
```

Comment:
I don't think this should be a warning, we use default quantization configs for the most of our models and have `quantization.yml` for those that couldn't be quantized with the default one. The warning in these cases may be confusing for the user. Probably just a check if `model_root` exists is enough.